draft-ietf-lamps-cms-ml-dsa.md

```diff
@@ -212,10 +214,6 @@ One method is used when signed attributes are present in the signedAttrs field o
 Each method produces a different "message digest" to be supplied to the signature algorithm in question, but because the pure mode of ML-DSA is used, the "message digest" is in fact the entire message.
 Use of signed attributes is preferred, but the conventions for signed-data without signed attributes is also described below for completeness.
 
-EDNOTE: Would it make sense to make a stronger statement here?
```

Comment:
I think no, at least to align with cms-sphincs-plus. The CMS-EUF-CMA work can address this.  If that determines we need more text here and in cms-sphincs-plus we can do that then.

Reply:
I know the WG wanted a quick turnaround on this draft - are we expecting it to wait on the CMS-EUF-CMA work (the interim for which will likely be in the new year)?

Reply:
I will post a summary of the EUF-CMA mitigations to the list since the presentation was cut short.  Hopefully responses to that will give an indication of where the WG wants to go from there.

cms-sphincs-plus has gone through WGLC, so I don't think the EUF-CMA stuff would prevent this draft from doing the same.  If the WG determines that the EUF-CMA issue needs to be solved immediately, both drafts would need to be updated.  But there is a more general EUF-CMA solution that also allows these drafts to proceed now.

Reply:
Makes sense to me. There are mitigations that can be applied in the interim for the EUF-CMA issue (e.g. only accepting one of signed attributes/no signed attributes, not both), so it's not like implementers have no options if they're worried about it prior to a separate RFC addressing the problem.

I'm happy with the text you've suggested - it could be worth noting the issue in the Security Considerations perhaps, and that future CMS RFCs may address this. On the other hand, I guess it's not an issue that's specific to ML-DSA.